dll/win32/iernonce/include/registry.h

```diff
@@ -0,0 +1,107 @@
+/*
+ * PROJECT:     ReactOS system libraries
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Functions to read RunOnceEx registry.
+ * COPYRIGHT:   Copyright 2021 He Yang <1160386205@qq.com>
+ */
+
+#pragma once
+
+#include <windows.h>
+#include <atlbase.h>
+#include <atlstr.h>
+#include <atlcoll.h>
+#include <atlsimpcoll.h>
+
+#define FLAGS_NO_STAT_DIALOG 0x00000080
```

Comment:
It may make sense to add more flags (for implementation in the future). You can see the list of flags here:  http://greatis.com/webhelp/regrun___detailed_instructions/start_control/runex_key.htm

Reply:
Nice reference @amber8706 , also it explains the reason of the "||" at the start of some runonceex entries.